provider/inspec.rb

```diff
@@ -60,6 +61,32 @@ def generate_resource(data)
       generate_properties(data.clone, data.object.all_user_properties)
     end
 
+    # Generate the IAM policy for this object. This is used to query and test
+    # IAM policies separately from the resource itself
+    def generate_iam_policy(data)
+      property_target = File.join(data.output_folder, 'libraries/google/iam/property')
+      FileUtils.mkpath property_target
+
+      FileUtils.cp_r 'templates/inspec/iam_policy/properties/.', property_target
```

Comment:
Could this be part of a common copy step for inspec instead of part of the generator code base? eg: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/provider/core.rb#L242